### ISSUE TITLE
Orientation to follow posture, not screen dimensions

### ISSUE THREAD

**Issue by verbunk** [FossifyOrg/Phone#184]
### Checklist

- [X] I made sure that there are **no existing issues** - [open](https://github.com/FossifyOrg/Phone/issues) or [closed](https://github.com/FossifyOrg/Phone/issues?q=is%3Aissue+is%3Aclosed) - to which I could contribute my information.
- [X] I made sure that there are **no existing discussions** - [open](https://github.com/FossifyOrg/Phone/discussions) or [closed](https://github.com/FossifyOrg/Phone/discussions?discussions_q=is%3Aclosed) - to which I could contribute my information.
- [X] I have read the FAQs inside the app (Menu -> About -> FAQs) and my problem isn't listed.
- [X] **I have taken the time to fill in all the required details. I understand that the bug report will be dismissed otherwise.**
- [X] This issue contains only one feature request.
- [X] I have read and understood the [contribution guidelines](https://github.com/FossifyOrg/Phone/blob/master/CONTRIBUTING.md).
- [X] I optionally [donated](https://fossify.org/donate) to support the Fossify mission.

### Feature description

Addition of a setting, 'Follow Posture', which changes the nature of screen rotations and app component sizing.

### Why do you want this feature?

While navigating through the app it will rotate itself to some preferred orientation per view without me rotating the phone. AFAICT this is based on screen to view size matching (closest aspect ratio?) b/w the contact list vs. dialer views. I would prefer the app not rotate and instead stay in the correct/upright orientation based on current phone posture.

**Comment by verbunk** [comment]
It's a twin-screen type. 1344w X 1892h. I just clicked around in every area I could find. As far as I can tell the phone in-call view is the only one that doesn't respond correctly. So no impact to Calendar / MMS / Contacts / Clock that I can see. That being said the Calendar has slightly awkward layout, it's not 50/50 making it split down the middle but that's a different repo / enhancement req. :)

**Comment by naveensingh** [comment]
>Addition of a setting, 'Follow Posture'

I don't think we need an option for this. It should respect the device orientation by default. The dialpad and call screens are currently hard coded to use portrait screen orientation, adding landscape layouts will fix the issue. This likely affects many other projects as many of the apps have [hardcoded](https://github.com/search?q=org%3AFossifyOrg+screenOrientation%3D%22&type=code) orientation in some places.
